virtcontainers/clh.go

```diff
@@ -410,6 +412,38 @@ func (clh *cloudHypervisor) getThreadIDs() (vcpuThreadIDs, error) {
 	return vcpuInfo, nil
 }
 
+func (clh *cloudHypervisor) hotplugBlockDevice(drive *config.BlockDrive) error {
```

Comment:
Could you try adding some unit tests for this new function using [`clhClientMock`](https://github.com/kata-containers/runtime/blob/master/virtcontainers/clh_test.go#L65)?

Reply:
Good point. I will ask inputs from @jcvenegas  and try to add one. :)